### DISCUSSION TITLE
How to do this with VirtualBox? I am unable to get Windows Sandbox.

### DISCUSSION THREAD

**Discussion by Marco12334** [gucci-on-fleek/lockdown-browser#42] [Q&A]
I do not have access to Windows Sandbox. And I am having trouble 'easily' making this work for other virtual machines mainly VirtualBox.

**Comment by gucci-on-fleek** [comment]
The easiest way to do this would be to enable “nested virtualisation” in VirtualBox, install Windows Pro inside of VirtualBox, then install and run the _Windows Sandbox_ inside of VirtualBox. But nested virtualisation is notoriously fickle, so even this might be quite tricky.

**Reply by Reizey** [reply]
will this work ? I am having the same issue
